util/checker.py

```diff
@@ -38,9 +39,16 @@ def get_data_symtab(self) -> dsym.DataSymbolContainer:
     def get_mismatch(self) -> (int, int, str):
         return self._mismatch_addr1, self._mismatch_addr2, self._mismatch_cause
 
+    def get_data_symbols_path(self, version = config.get_default_version()) -> Path:
+        value = utils.get_repo_root() / 'data'
```

Comment:
Can we add a "get data root" utility function? I've seen this pattern several times by now.